packages/happy-dom/src/form-data/FormData.ts

```diff
@@ -25,12 +28,45 @@ export default class FormData implements Iterable<[string, string | File]> {
 	 * Constructor.
 	 *
 	 * @param [form] Form.
+	 * @param [submitter] A submit button that is a member of the form.
 	 */
-	constructor(form?: HTMLFormElement) {
+	constructor(form?: HTMLFormElement, submitter?: HTMLInputElement | HTMLButtonElement) {
 		if (!form) {
 			return;
 		}
 
+		if (submitter && submitter instanceof HTMLElement) {
```

Comment:
Looking at JSDOM, it seems like it should be implemented a bit different. We will have to look into the spec a bit more.

Link to JSDOM implementation:
https://github.com/jsdom/jsdom/blob/main/lib/jsdom/living/xhr/FormData-impl.js